### ISSUE TITLE
Ogmios Health Status stays always %99.9999

### ISSUE THREAD

**Issue by miracatici** [CardanoSolutions/ogmios#161]
Hello there, I'm using cardano-node-ogmios v4.2.1 with Docker image, alongside IOHK Wallet backend and cardano-db-sync Docker images. I'm using: 1337 port to check health status manually. It states always %99.999 even after fully synced. I can manage my wallets, follow the chain with db-sync and any other methods,  make transactions with wallet service. It seems to be working successfully.

Is there any problem with percentage?

**Comment by KtorZ** [comment]
The network synchronization is an approximation of how far your local node is from the _projected tip of the network_.

The tip of the network is calculated from era parameters, the current time and the last block known of your local node. It has a resolution of 5 digits, which means that currently, the smallest possible increment (0.001%) represent about 20minutes of time (modulo rounding, this may be half that, I don't recall exactly). 

Thus, if your node is mostly synchronized but still about 20 minutes from the actual network tip, then it'll not show 100%. This could be explained a various ways, for example, the clock on your machine is out of sync and reports a time that isn't the actual one. Ogmios doesn't do any NTP check to make sure of that, maybe it should. 

Do you have an example with:

- The last known tip (reported by the health status)
- The network synchronization (also reported by the health status)
- The current time of your machine

To see if indeed, there's something abnormal here?

**Comment by miracatici** [comment]
I don't know that there is a near-realtime online tip number check page, I'm checking various sources such as `cardanoscan` `cardano explorer` `pool.pm` or any other simultaneously; my local node almost realtime update  slot block tip numbers. There is no lag or abnormal activity. Maybe the current time and time synchronization occurs this problem. I will check it.

**Comment by miracatici** [comment]
Hello, in this duration I checked many things like local date-time, my mainboard battery etc. I also set up a testnet node. Surprisingly, the dashboard of testnet node reports %100 OK "more often". I said "more often" because after full-sync it reports most of the time OK, but not always.

I made a lot of transactions, add/create/delete wallets with cardano-wallet, and sync them to be ready. No abnormal activity occurred. I just think that is a simple lagging and rounding issue.
